### ISSUE TITLE
[4.0] installer script as vanilla, es6

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#21076]
new installer script from #21034 does not work

### Steps to reproduce the issue
try to drag and drop install an extension in the extension installer

### Expected result
extension installs

### Actual result
spinner never stops and errors in the console
```

installer.min.js:1 POST http://127.0.0.1/cms4/administrator/index.php?option=com_installer&task=install.ajax_upload 500 (Internal Server Error)
(anonymous) @ installer.min.js:1
installer.min.js:1 Uncaught ReferenceError: options is not defined
    at XMLHttpRequest.s.onreadystatechange (installer.min.js:1)
s.onreadystatechange @ installer.min.js:1
XMLHttpRequest.send (async)
(anonymous) @ installer.min.js:1


```

**Comment by dgrammatiko** [comment]
@brianteeman can you check #21077

**Comment by ghost** [comment]
closed as having Pull Request #21077
